src/calculateRentalCost.js

```diff
@@ -4,7 +4,15 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  let sale = 0;
```

Comment:
Prefer `const` over `let` for variables that are not reassigned. In this case, `sale` does not need to be reassigned if you return directly from within the `if` blocks.